lib/public/views/Logs/Create/index.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+import { h } from '/js/src/index.js';
+
+/**
+ * A function to construct the create log screen
+ * @param {object} model Pass the model to access the defined functions
+ * @return {vnode} Return the view of the inputs
+ */
+const createScreen = (model) => {
+    const title = model.logs.getTitle();
+    const text = model.logs.getText();
+    const disabled = !(title.length > 2 && text.length > 2);
+    const error = model.logs.hasErrorOccured();
+    const errorMessage = model.logs.getErrorMessage();
+
+    return h('div#create-log', [
+        h('', [
+            h('h3.black.line-break: auto', 'Title of the log'),
+            h('input#title', {
+                placeholder: 'Enter the title of the log entry...',
+                minlength: 3,
+                maxlength: 140,
+                value: title,
+                oninput: (e) => model.logs.setTitle(e.target.value),
+            }, title),
+        ]),
+        h('', [
+            h('h3.black.line-break: auto', 'Text'),
+            h('textarea#text.w-75.form-control', {
+                placeholder: 'Your message...',
+                minlength: 3,
+                rows: 5,
+                oninput: (e) => model.logs.setText(e.target.value),
+            }, text),
+        ]),
+        h('button.btn.btn-success.m#send', {
+            onclick: () => model.logs.createLog(),
+            disabled,
+        }, 'Create'),
+        error &&
+            h('span.danger.mh2#error', errorMessage || 'There was a problem with posting the log, try again later'),
```

Comment:
I would expect the errors to be the item at the top of the screen, also have a look at [Bootstrap alerts](https://getbootstrap.com/docs/4.0/components/alerts/#dismissing).

Reply:
This one as well

Reply:
![image](https://user-images.githubusercontent.com/25134477/82657069-02a03c00-9c25-11ea-803d-c7837e815f53.png)

Errors are being chained while I would expect them to be on a newline

Reply:
The lines are breaking properly now